src/quo2/components/profile/select_profile/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo2.components.profile.select-profile.view
+  (:require
+    [quo2.components.profile.select-profile.style :as style]
+    [react-native.core :as rn]
+    [quo2.components.markdown.text :as text]
+    [quo2.components.avatars.user-avatar :as user-avatar]
+    [reagent.core :as reagent]))
+
+(defn- on-change-handler
+  [selected? on-change]
+  (swap! selected? not)
+  (when on-change (on-change @selected?)))
+
+(defn view
+  "Options
+   - `default-selected?` The default selected state. Use when the component is not controlled.
+   - `selected?` Selected state. Use when the component is controlled.
+   - `name` Full name
+   - `profile-picture` Profile picture
+   - `customization-color` Customization color
+   - `(on-change selected?)` On change event handler 
+   "
+  [{:keys [default-selected?]}]
+  (let [internal-selected? (reagent/atom (or default-selected? false))]
+    (fn [{:keys [selected?
+                 profile-picture
+                 name
+                 customization-color
+                 on-change]
+          :or   {customization-color :turquoise}}]
+      (when (and (not (nil? selected?)) (not= @internal-selected? selected?))
+        (reset! internal-selected? selected?))
+      [rn/touchable-opacity
+       {:style               (style/container customization-color @internal-selected?)
+        :on-press            #(on-change-handler internal-selected? on-change)
+        :active-opacity      1
+        :accessibility-label :select-profile}
+       [rn/view {:style style/header}
+        [user-avatar/user-avatar
+         {:full-name         name
+          :profile-picture   profile-picture
+          :status-indicator? false
+          :ring?             true
+          :size              :medium}]
+        [rn/view {:style (style/select-radio @internal-selected?)}
+         (when @internal-selected? [rn/view {:style style/select-radio-inner}])]
+       ]
```

Comment:
Please move closing bracket `]` to above line

Reply:
My bad. fixed it.

Reply:
Hi, sorry to bother again, Please can you also remove this trailing space
![image](https://user-images.githubusercontent.com/17097240/220728364-e36778ad-6bd9-45f7-988b-0dca31db762f.png)

Reply:
Np. I have removed that space.

Is it possible to add these checks to lint?

Reply:
cc @yqrashawn